### PR TITLE
Bump version to 0.3.6

### PR DESCRIPTION
**Pull request by kaimallea**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "isMobile",
-  "version": "0.3.5",
+  "version": "0.3.6",
```
Comment (shinnn) [inline bower.json]: `version` is no longer supported by bower.

https://github.com/bower/bower.json-spec#version

**Comment by kaimallea** [comment]
yikes, thanks. reverted it @shinnn

**Comment by kaimallea** [comment]
I'll prob just revert the revert and leave version in there, as it's just ignored. I'll create the git tag, though.

**Pull request merged**